infra/terraform/prod/vpc.tf

```diff
@@ -39,11 +46,11 @@ resource "aws_internet_gateway" "wikijump_igw" {
 
 # Routes
 
-# # TODO: Add as needed and as will improve security posture.
+
```

Comment:
Why was this comment removed? Should there still be a `Routes` section?

Reply:
I didn't initially have any routing set up in here, what's available now is what's needed for the environment to run and at the moment I'm okay with dev running on a publicly available instance to have another way in aside from via the load balancer. If in the future the load balancer piece is so rock solid that I want to put the container EC2 in a private subnet the routes will change.